src/allmydata/scripts/tahoe_start.py

```diff
@@ -104,7 +104,7 @@ def start(config):
     # "pretty fast" and with a zero return-code, or else something
     # Very Bad has happened.
     try:
-        args = [sys.executable]
+	args = [sys.executable] if not getattr(sys, 'frozen', False) else []
```

Comment:
this looks like it changes the indentation? or is that an artifact of github's diff viewer?

Reply:
Nice catch -- not sure where that tab came from.. Fixed now.